### DISCUSSION TITLE
关于设置选项或右键菜单的建议

### DISCUSSION THREAD

**Discussion by iam6668** [NagaseIori/DyNode#72] [Ideas]
可以添加关于和制作名单选项，算是将版本和署名写上去了，最好把反馈按钮也加上
想说点私心的话，可以的话在把我加上吧，毕竟我也给了不少建议吧，当然不加也行，但一定记得把作者名字加上（没错就是你，这项目现在就我们还关注一下）

**Comment by NordLandeW** [comment]
很感谢你的建议，不过原则上 Contributors 的列表需要你对项目本身做出改动或贡献才会加入 w

署名是有的，不过比较隐蔽，F11打开调试信息就能看到

所有的感谢名单、许可与版权信息都放在官网的关于部分。

说实话在制作DyNode的时候我就已经做好了完全无人关注的心理准备，对DyNode的后续更新也只是出于对项目本身的责任心；DyNode 收到的反馈比较少的原因也有使用的大家可能访问 Github 存在困难或不会使用的可能，何况我已经从一些使用者和谱师（如 Jmaks）那里得到了十分有用的帮助，我十分感谢他们。

不一定要将反馈当做支持的义务，而且大家肯定有更重要的事情去做。使用 DyNode 项目本身就是对 DyNode 项目（以及我）的最大支持w。反馈能使大家的使用体验变得更好，因此我也十分支持大家在觉得有必要的情况下进行反馈。

至于为什么项目页面和文档让我在上面花了这么多时间，说实话也没有特别认真的理由，features 也都是我乱写的，大概很好看出来

但总之很感谢你关注该项目，之后也欢迎你在你觉得合适的范围内继续支持此项目 w

**Reply by iam6668** [reply]
这个呢反馈就是顺手的事，不用太在意，有些我没看到的也感谢提醒了
这边呢不知道怎么联系你，我这边能把dynode的录制视频放在破站吗？

**Reply by NordLandeW** [reply]
> 这个呢反馈就是顺手的事，不用太在意，有些我没看到的也感谢提醒了 这边呢不知道怎么联系你，我这边能把dynode的录制视频放在破站吗？

如果只是bug的反馈视频可以直接提交到github的对应issue页面上

DyNode 基于 MIT 协议进行许可，没有对使用/改编等方式做出任何限制，所以请放心录制视频

**Reply by iam6668** [reply]
等等，，这条你看到了给个答复我就删，pad，mixer，mutil侧的降落音符限制在日程上吗，没有就赶紧建一个issue

**Reply by NordLandeW** [reply]
> 等等，，这条你看到了给个答复我就删，pad，mixer，mutil侧的降落音符限制在日程上吗，没有就赶紧建一个issue

见 #7
